### DISCUSSION TITLE
Having issues with private fields on older browsers

### DISCUSSION THREAD

**Discussion by neokry** [wevm/wagmi#241] [Q&A]
I can see some wagmi classes are using private fields and this is causing issues for users with older browsers that don't natively support this. Tried settings up custom babel plugins for this but those aren't working either. Using the latest nextjs version for this. Anyone have a solution?

**Comment by tmm** [comment]
Any idea what browsers these folks are using? Support [seems](https://caniuse.com/mdn-javascript_classes_private_class_fields) [good](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#browser_compatibility), but down to make changes if necessary.

**Reply by neokry** [reply]
Im seeing issues in safari 14.0.3 on osx. Users have mentioned mobile browsers are sometimes causing issues guessing its from safari on non updated iphones or something

**Reply by neokry** [reply]
Safari on IOS 14.8 confirmed to have issues

**Reply by tmm** [reply]
Should be fixed now in `0.2.23`.
